website/pages/en/sps/triggers-example.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Example Susbtreams Trigger
+---
+
+Here you’ll walk through a Solana based example for setting up your Substreams-powered subgraph project. If you haven’t already, first check out the [Getting Started Guide](https://github.com/streamingfast/substreams/blob/enol/how-to-guides/docs/new/how-to-guides/intro-how-to-guides.md) for more information on how to initialize your project.
+
+Consider the following example of a Substreams manifest (`substreams.yaml`), a configuration file similar to the `subgraph.yaml`, using the SPL token program Id:
+
+```graphql
+specVersion: v0.1.0
+package:
+  name: my_project_sol
+  version: v0.1.0
+
+imports: #Pass your spkg of interest
+  solana: https://github.com/streamingfast/substreams-solana-spl-token/raw/master/tokens/solana-spl-token-v0.1.0.spkg
+
+modules:
+
+ - name: map_spl_transfers
+   use: solana:map_block #Select corresponding modules available within your spkg
+   initialBlock: 260000082
+
+ - name: map_transactions_by_programid
+   use: solana:solana:transactions_by_programid_without_votes
+
+network: solana-mainnet-beta
+
+params: #Modify the param fields to meet your needs
+			  #For program_id: TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
+  map_spl_transfers: token_contract:orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE
+```
+
+Now see the corresponding subgraph manifest **(`subgraph.yaml`)** using a Substreams package as the data source:
```

Comment:
```suggestion
Now see the corresponding subgraph manifest (`subgraph.yaml`) using a Substreams package as the data source:
```
(inline code cannot be made bold, so this is effectively just making the parentheses bold)